modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/form21a_spec.rb

```diff
@@ -5,6 +5,19 @@
 RSpec.describe 'AccreditedRepresentativePortal::V0::Form21a', type: :request do
   let(:valid_json) { { field: 'value' }.to_json }
   let(:invalid_json) { 'invalid json' }
+  let(:schema) do
+    {
+      '$schema' => 'http://json-schema.org/draft-04/schema#',
+      'title' => 'Apply to become a VA-accredited attorney or claims agent',
+      'type' => 'object',
+      'properties' => {
+        'firstName' => {
+          'type' => 'string'
+        }
+      }
+    }
+  end
+  let(:wrong_schema) { { 'firstName' => 1234 }.to_json }
```

Comment:
This isn't a schema but rather the form so we could call it `invalid_form` or something.